dev/com.ibm.ws.ui.tool.serverConfig/bnd.bnd

```diff
@@ -0,0 +1,35 @@
+#*******************************************************************************
+# Copyright (c) 2016, 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: WebSphere Admin Center Server Config Tool
+Bundle-SymbolicName: com.ibm.ws.ui.tool.serverConfig
+Bundle-Description: WebSphere Admin Center Server Config Tool, version ${bVersion}
+
+Web-ContextPath: /ibm/adminCenter/serverConfig-1.0
+OL-VirtualHost: ${admin.virtual.host}
+
+IBM-Authorization-Roles: com.ibm.ws.management
+
+Include-Resource: \
+  ${if;${driver;gradle};lib/orion_editor=lib/orion}, \
```

Comment:
So this bundle builds differently if being run from the command line vs from eclipse or something else?

Reply:
I believe this is to make things work well in eclipse without having to run gradle in order to build in eclipse.